shop-info-service/handlers/createProduct.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { Client } from 'pg';
+
+import getSampleResponse from '../helpers/getSampleResponse';
+import getErrorResponse from "../helpers/getErrorResponse";
+import dbOptions from "../helpers/getDbOptions";
+
+
+export const createProduct: APIGatewayProxyHandler = async (event) => {
+  const {title, description, price, count} = JSON.parse(event.body);
+  const client = new Client(dbOptions);
+  await client.connect();
+  console.log(`event ${JSON.stringify(event)}`, `body ${JSON.stringify(event.body)}`);
+
+  try {
+    await client.query('BEGIN');
+    const {rows: [{id: insertedProductId}]} = await client.query('insert into product_list (title, description, price) values ($1, $2, $3) returning id', [title, description, price]);
+    await client.query('insert into stock_list (product_id, count) values ($1, $2)', [insertedProductId, count]);
+    const {rows: products} = await client.query(
+      'select p.id, p.description, p.title, p.price, p.img_url as image, ' +
```

Comment:
You can use backticks ( ` ) for multiline string

Reply:
ok, thank you)